source/move.js

```diff
@@ -0,0 +1,30 @@
+import _curry3 from './internal/_curry3';
+
+/**
+ * Move an item, at index `from`, to index `to`, in a list of elements.
+ * A new list will be created containing the new elements order.
+ *
+ * @func
+ * @memberOf R
+ * @category List
+ * @sig Number -> Number -> [a] -> [a]
+ * @param {Number} from The source index
+ * @param {Number} to The destination index
+ * @param {Array} list The list which will serve to realise the move
+ * @return {Array} The new list reordered
+ * @example
+ *
+ *      R.move(0, 2, [1,2,3]); //=> [2,3,1]
+ *      R.move(-1, 0, [1,2,3]); //=> [3,1,2] list rotation
```

Comment:
I think `[1, 2, 3]` could too easily cause confusion, especially given how close it is to indices.  I also think it's too short by at least one.  Either `['a', 'b', 'c', 'd']` or `[8, 6, 7, 5, 3, 0, 9]` would demonstrate things better.